### PR TITLE
Changed max inline player height

### PR DESCRIPTION
**Pull request by DeepDoge**
Update to this PR https://github.com/OdyseeTeam/odysee-frontend/pull/505
After many of the CSS has been changed recently.

### CODE REVIEW

**Comment by DeepDoge** [comment]
Ok, this makes it hide like/dislike buttons on small screens when view counter and action buttons doesn't fit together to the row.
Without this change like/dislike buttons are at least still visible on smaller screens.

I would make some changes to make it work but it would look like a patch.
<details>
  <summary>The problem is</summary>
I believe, the whole CSS is outdated.<br/>
Everything is placed and set very precisely and not relative to each other.<br/>
Basically, changing one thing breaks everything.<br/>
Maybe whole CSS needs to be rewritten while not changing the look of the site.<br/>
But who is gonna do that, too much work. 
</details>
